misc/upload_release.py

```diff
@@ -45,9 +45,11 @@ def upload_artifact(is_taichi):
         sys.exit(f'Missing password env var {pwd_env}')
     command = [sys.executable, '-m', 'twine', 'upload']
     if not is_taichi:
-        command.extend(['--repository', 'testpypi'])
-    command.extend(
-        ['--verbose', '-u', '__token__', '-p', twine_password, 'dist/*'])
+        command.extend(
+            ['--repository-url', 'https://pypi.taichi.graphics/simple/'])
+    uname = '__token__' if is_taichi else 'admin'
```

Comment:
Does pypicloud supports tokens? If so, we should better use a token.

Reply:
@frostming: @rexwangcc and I just checked that Pypi Cloud seems not have access token support (there is registration token though, which does not help). Should we just stick to the user name login?

Reply:
Based on what we have learned from the pypicloud setup, I suggest we stick the basic auth login approach (of course we are using a non-admin account for uploading) :P

Reply:
```suggestion
    uname = '__token__' if is_taichi else 'taichi-nightly-cloud-pypi-uploader'
```